src/main/java/uk/gov/justice/hmiprobation/casesampler/services/AllocationCalculator.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.hmiprobation.casesampler.services
+
+import uk.gov.justice.hmiprobation.casesampler.dto.Case
+import uk.gov.justice.hmiprobation.casesampler.utils.Size
+import uk.gov.justice.hmiprobation.casesampler.utils.Size.SampleSize
+import uk.gov.justice.hmiprobation.casesampler.utils.calculateSampleSize
+import kotlin.random.Random
+
+data class Info(val id: String, val size: Size)
+
+data class AllocationData(val cluster: Info, val ldu: Info, val ro: Info)
+
+data class RoAllocation(val allocationData: AllocationData, val cases: List<Case>) {
```

Comment:
we return information about how the number of cases for a specific RO was calculated.